splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -946,5 +945,15 @@ public ColPermsDescriptor getColumnPermissions(UUID tableUUID,
         return manager.isEnabled()?manager.getColPermsManager().getColumnPermissions(this,tableUUID,privType,forGrant,authorizationId):null;
```

Comment:
A minor comment here, since we have the function isEEManagerEnabled() now, we could use this function instead of repeating the same code.